x-pack/plugins/reporting/server/lib/tasks/execute_report.test.ts

```diff
@@ -99,6 +99,7 @@ describe('Execute Report Task', () => {
     const store = await mockReporting.getStore();
     store.setReportError = jest.fn(() => Promise.resolve({} as any));
     const task = new ExecuteReportTask(mockReporting, configType, logger);
+    // @ts-expect-error private method
     task._claimJob = jest.fn(() =>
```

Comment:
22cb23d7632

Reply:
@vadimkibana The error checking is disabled on this line because `_claimJob` is a private method which shouldn't be accessed from the outside. But I am accessing it for testing purposes.

Hope that explanation helps.